sycl/doc/extensions/experimental/sycl_ext_matrix/sycl_ext_intel_matrix.asciidoc

```diff
@@ -198,6 +198,11 @@ joint_matrix_apply(sg, A, [=](T &val, size_t row, size_t  col) {
    sum_local_rows[row] += val;
 });
 ```
+
+In the case of `ext_intel_packed` matrix memory layout, `row` and
```

Comment:
I believe this should be also reflected in SPIR-V specification for coordinate instruction. @dkhaldi @Nuullll

Reply:
I agree